### DISCUSSION TITLE
Cannot get multiple node-red Sensors to show in HA

### DISCUSSION THREAD

**Discussion by JohnREstes** [zachowj/node-red-contrib-home-assistant-websocket#1699] [Q&A] [answered]
I have followed the instructions, and set every thing up.  When I had one flow, all was well, I added 5 more, and cannot figure out how to find/see in HA.  I am sure I missed something.  I just don't know how to get them to show!   PLEASE Help

<img width="599" alt="Screenshot 2024-12-01 at 10 57 50 PM" src="https://github.com/user-attachments/assets/d41b94bf-c50e-4e6d-af52-734c85d30c8b">
<img width="1260" alt="Screenshot 2024-12-01 at 10 59 00 PM" src="https://github.com/user-attachments/assets/5597a5d6-d5ce-4218-b622-25db75ef7b16">
<img width="827" alt="Screenshot 2024-12-01 at 10 57 08 PM" src="https://github.com/user-attachments/assets/6c10baf9-5288-4c4e-b216-e8aa74dce1ff">
<img width="592" alt="Screenshot 2024-12-01 at 10 57 40 PM" src="https://github.com/user-attachments/assets/1209e8e2-d498-4fc7-b163-262cf957c4be">
<img width="632" alt="Screenshot 2024-11-30 at 11 32 48 PM" src="https://github.com/user-attachments/assets/739fe334-14cb-4b1e-9f40-ce12de219676">
<img width="539" alt="Screenshot 2024-12-01 at 10 57 28 PM" src="https://github.com/user-attachments/assets/93a57d2e-bf07-4cb3-900c-fca95b260422">

**Comment by zachowj** [comment, marked as answer]
Are you using the same entity configuration for all six sensor nodes? If so, you'll need a separate entity configuration node for each sensor you want to create in Home Assistant.

**Reply by JohnREstes** [reply]
You are brilliant!  I knew I was missing something!  Worked like a champ!   THANKS!  I will work on units next!

<img width="439" alt="Screenshot 2024-12-01 at 11 17 20 PM" src="https://github.com/user-attachments/assets/6d852356-468d-4c1a-8e26-499ea7aeb55e">
